rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -11,30 +11,28 @@
 from rotkehlchen.fval import FVal
 from rotkehlchen.history.events.structures.evm_event import EvmEvent
 from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
-from rotkehlchen.tests.utils.accounting import (
-    TIMESTAMP_0_MS,
-    TIMESTAMP_0_SEC,
-    TIMESTAMP_1_MS,
-    TIMESTAMP_1_SEC,
-    accounting_history_process,
-    assert_pnl_totals_close,
-)
+from rotkehlchen.tests.utils.accounting import accounting_history_process, assert_pnl_totals_close
 from rotkehlchen.tests.utils.factories import make_evm_address, make_evm_tx_hash
 from rotkehlchen.tests.utils.messages import no_message_errors
-from rotkehlchen.types import Location, Price
+from rotkehlchen.types import Location, Price, Timestamp, TimestampMS
 
 if TYPE_CHECKING:
     from rotkehlchen.accounting.accountant import Accountant
 
+TIMESTAMP_1700000000_SEC = Timestamp(1700000000)
```

Comment:
There is no need to define constants for these, or name the constant with the value of the timestamp. Come on wtf?\

If you really want to define them call them `TIMESTAMP_A`and `TIMESTAMP_B`